src/Providers/ZendeskServiceProvider.php

```diff
@@ -29,16 +30,22 @@ public function register() {
         $this->publishes([
             $configPath => config_path(sprintf('%s.php', $packageName)),
         ]);
-	}
-
-	/**
-	 * Bind service to 'zendesk' for use with Facade
-	 *
-	 * @return void
-	 */
-	public function boot()
-	{
-        $this->app->bind('zendesk', ZendeskService::class);
-	}
-
-}
\ No newline at end of file
+    }
+
+    /**
+     * Bind service to 'zendesk' for use with Facade.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        $this->app->bind('zendesk', function () {
+            $driver = config('newsletter.driver', 'api');
```

Comment:
@imacrayon I'm guessing this is meant to reference the zendesk-laravel config and not the newsletter?

Reply:
Fixed